src/assets/sass/_header.scss

```diff
@@ -0,0 +1,109 @@
+@media only screen and (min-width: 769px){
+    .header-container {
+        display: block;
+    }
+}
+.header-container .header-main {
+    color: #0b0c0c;
+    padding: 0.5rem 0px;
+    border-bottom: 6px solid #008938;
+    background: #FFF !important;
+}
+
+.header-div-container {
+    width: 100%;
+}
+
+.header-container .header-main .site-width-container {
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+}
+
+@media only screen and (min-width: 1020px) {
+    .header-div-container .site-width-container {
+        width: 100%;
+        margin: 0px auto;
+        max-width: 960px;
+        padding: 0px;
+    }
+}
+.header-div-container .site-width-container {
+    display: flex;
+    flex-direction: row;
+    width: auto;
+    padding: 0px 16px;
+}
+
+.header-logo-container {
+    display: flex;
+    -webkit-box-align: center;
+    align-items: center;
+}
+
+.header-logo-container .defra-logo {
+    padding-left: 2px;
+    border-left: 2px solid #00703c;
+}
+
+.header-logo-container a, .header-logo-container a:visited, .header-logo-container a:link {
+    margin-left: 0.5rem;
+    color: #0b0c0c;
+    text-decoration: none;
+}
+
+.logo-link:link, .logo-link:visited {
+    color: #008938;
+}
+
+.header-logo-link:link {
+    color: rgb(29, 112, 184);
```

Comment:
Is this necessary? The same applies for `.logo-link:link, .logo-link:visited`

Reply:
agreed, we dont need this. removed